packages/components/Button/src/ButtonTokens.ts

```diff
@@ -9,7 +10,10 @@ export const defaultButtonTokens: TokenSettings<ButtonTokens, Theme> = () =>
       width: '100%',
     },
     medium: {
-      padding: globalTokens.spacing.sNudge - globalTokens.stroke.width.thin,
+      padding: Platform.select({
+        ios: 6,
+        default: globalTokens.spacing.sNudge - globalTokens.stroke.width.thin,
+      }),
```

Comment:
Can we move this to a ButtonTokens.ios.ts file?

Reply:
Or is this just here for testing and I was too quick 😅

Reply:
This is a temporary patch needed just to get the tester stood up without errors, I copied what Android is doing from https://github.com/microsoft/fluentui-react-native/pull/2257. I'm not sure that we would still need a .ios.ts or .android.ts file once a better fix is added (maybe we're removing sNudge altogether?), so a little hesitant to add it now if we're just going to delete it later

Reply:
When we were deciding how to put platform specific control tokes, we decided to use platform specific files for one main reasons: The overall bundle is smaller, since the other platform files are filtered out. This isn't true if you use platform select. 

There might have been other reasons, but I can't remember them. So as a rule, I try to have control tokens actually be separated into different platform files.